src/app/fdctl/run/tiles/fd_repair.c

```diff
@@ -537,6 +553,17 @@ unprivileged_init( fd_topo_t *      topo,
   fd_net_create_packet_header_template( ctx->intake_hdr, FD_REPAIR_MAX_PACKET_SIZE, ctx->repair_intake_addr.addr, ctx->src_mac_addr, ctx->repair_intake_listen_port );
   fd_net_create_packet_header_template( ctx->serve_hdr, FD_REPAIR_MAX_PACKET_SIZE, ctx->repair_serve_addr.addr, ctx->src_mac_addr, ctx->repair_serve_listen_port );
 
+  /* Keyguard setup */
+  fd_topo_link_t * sign_in = &topo->links[ tile->in_link_id[ SIGN_IN_IDX ] ];
+  fd_topo_link_t * sign_out = &topo->links[ tile->out_link_id[ SIGN_OUT_IDX ] ];
+  if ( fd_keyguard_client_join( fd_keyguard_client_new( ctx->keyguard_client,
```

Comment:
```suggestion
  if( fd_keyguard_client_join( fd_keyguard_client_new( ctx->keyguard_client,
```

See `contrib/lint.sh`